yurt/yurt_core/setup.py

```diff
@@ -166,35 +136,20 @@ def copy_pem_file(user, host, key_name):
 
 
 @setup.command()
-@click.option('--git_repo', default=None, help='Git Repo Link')
-@click.option('--skip_vagrant', is_flag=True, help='Skip `vagrant up` at the end of program')
-@click.option('--vault', is_flag=True, help="Uses vault for git keys")
-def new_project(git_repo, skip_vagrant, vault):
+def new():
     """
     Create new project
     """
-    if git_repo is None:
-        try:
-            git_repo = raw_input("".join(("Enter the git repository link\n",
-                                          "(i.e. git@github.com:mr_programmer/robot_repository.git):\t")))
-        except NameError:
-            git_repo = input("".join(("Enter the git repository link\n",
-                                      "(i.e. git@github.com:mr_programmer/robot_repository.git):\t")))
-
-    ordered_methods = [
-        enable_git_repo,
-        create_project,
-        load_orchestration_and_requirements,
-        move_vagrantfile_to_project_dir,
-        copy_ansible_configs_to_parent,
-        add_all_files_to_git_repo
-    ]
-    args = create_settings(vault, git_repo)
-    # TODO: Validate `args` before running them through THE GAUNTLET
-    for method in ordered_methods:
-        method(*args)
-    if not skip_vagrant:
-        run('vagrant up')
+    template_location = 'gh:yeti/yurt_template-django'
+    print('==> Creating a new project with Yurt Django 2.0 Template')
+    cookiecutter(template_location)
+    print('==> Project created: {}'.format(
+        os.path.join(
+            os.getcwd(),
+            '<project_name>'
+        )
+    ))
+    print('==> cd to the above path and run `docker-compose up`︎')
```

Comment:
Idea with v2 to go for a more simple less automagical approach? We could potentially script this?

Reply:
I wanted to make as few assumptions as possible about the user's existing environment. Running a script would cause an error state if they haven't installed Docker + friends yet.

I could make wrappers for common Docker commands though if that's what you mean, but that would have to come at a later time.